config/schema/socrata.schema.yml

```diff
@@ -1,9 +1,11 @@
 socrata.settings:
-  type: mapping
```

Comment:
It was my understanding that this YAML was for a configuration object, which has a subtype that should be used

> There are two types that configuration schema predefines for configuration files: config_object for global configuration files and config_entity for entities.

https://www.drupal.org/docs/8/api/configuration-api/configuration-schemametadata